src/options.html

```diff
@@ -9,11 +9,11 @@
 </head>
 
 <body id="options-body">
-  <div id="container" class="grid-with-banner">
+  <div id="container" class="regular-grid">
     <!-- Main Tab Content + Donation plea -->
     <header>
       <div class="d-flex flex-column">
-        <div id="header-banner" class="info-banner d-flex flex-column">
+        <div id="header-banner" hidden class="info-banner d-flex flex-column">
```

Comment:
This can't be right, can it? 

Perhaps you meant:

```suggestion
        <div id="header-banner" class="info-banner d-flex flex-column hidden">
```

?

Reply:
The extension uses Bootstrap v4.3.1, and the `hidden` attribute seems to be fully supported:

https://getbootstrap.com/docs/4.1/content/reboot/#html5-hidden-attribute

Reply:
OMG, I had no idea. They did it, they really did.